rtpsniff.c

```diff
@@ -47,26 +47,28 @@ int main(int argc, char const *const *argv) {
 
     /* Try initialization */
     errbuf[0] = '\0';
-    strncat(errbuf, "Not enough arguments", PCAP_ERRBUF_SIZE - 1);
-    if (argc != 4 ||
-	    ((handle = pcap_create(argv[1], errbuf)) == NULL) ||
-	    (pcap_set_snaplen(handle, sniff_snaplen()) != 0) ||
-	    (pcap_set_timeout(handle, 1000) != 0) ||
+    if (argc != 4) {
+	strncat(errbuf, "Not enough arguments", PCAP_ERRBUF_SIZE - 1);
+	exit_error(errbuf);
```

Comment:
Hier had je (als exit_error een const-char had genomen) rechtstreeks de "Not enough..." string heen kunnen gooien, zonder strncat.